README.md

```diff
@@ -26,3 +26,147 @@ Our application would allow students to intuitively choose the best schedule for
 ## [Even Skari](https://github.com/evenskari)
 
 <img width="300" height="300" src="https://avatars2.githubusercontent.com/u/11603089">
+
+# Getting started
+First install Yarn from [here](https://yarnpkg.com/lang/en/docs/install/)<br/>  
+Find your workspace folder and clone the repository
+```
+git clone https://github.com/scalableinternetservices/ucsb_class_scheduler.git
+```
+
+
+While being in the newly cloned directory, configure and update gems by running
+```
+yarn install
```

Comment:
Aren't we using `bundle install` to install gems?

Reply:
Changed to '''bundle install'''